utils.js

```diff
@@ -1,15 +1,69 @@
 const { not, equals, pick } = require('ramda')
 
-async function createTable({ dynamodb, name, attributeDefinitions, keySchema }) {
+const validate = {
+  streamViewType: (inputs) => {
+    if (!inputs.streamViewType) {
+      return
+    }
+
+    const validStreamTypes = ['NEW_IMAGE', 'OLD_IMAGE', 'NEW_AND_OLD_IMAGES', 'KEYS_ONLY']
+    if (!validStreamTypes.includes(inputs.streamViewType)) {
+      throw Error(`${inputs.streamViewType} is not a valid streamViewType.`)
+    }
+  },
+
+  streamViewTypeUpdate: (comp, previousTable, inputs) => {
+    if (!previousTable.streamArn || !inputs.streamViewType) {
+      return
+    }
+
+    if (comp.state.stream && inputs.stream && comp.state.streamViewType !== inputs.streamViewType) {
+      throw Error(`You cannot change the view type of an existing DynamoDB stream.`)
+    }
+  }
+}
```

Comment:
I added validation around stream types for creating and updating tables. 
`streamViewType` just confirms they are using a correct view type, and will throw and error if they are not

`streamViewTypeUpdate` is a bit more subtle. If you change the viewType on an table that already has a stream, it wont do anything. Its possible people might assume it all worked unless they see a message explaining nothing actually happened. So I made an error for it.

@eahefnawy is throwing errors the best way to give messages like this in the CLI?